examples/smbpasswd.py

```diff
@@ -122,12 +126,22 @@ def parse_args():
 	parser.add_argument('target', action='store', help='[[domain/]username[:password]@]<targetName or address>')
 	parser.add_argument('-ts', action='store_true', help='adds timestamp to every logging output')
 	parser.add_argument('-debug', action='store_true', help='turn DEBUG output ON')
+
 	group = parser.add_mutually_exclusive_group()
 	group.add_argument('-newpass', action='store', default=None, help='new SMB password')
-	group.add_argument('-newhashes', action='store', default=None, metavar='LMHASH:NTHASH', help='new NTLM hashes, format is LMHASH:NTHASH '
-                                                                           '(the user will be asked to change their password at next logon)')
+	group.add_argument('-newhashes', action='store', default=None, metavar='LMHASH:NTHASH', help='new NTLM hashes, format is LMHASH:NTHASH (the user will be asked to change their password at next logon)')
+	group = parser.add_mutually_exclusive_group()
+	group.add_argument('-oldpass', action='store', default=None, help='old SMB password')
+	group.add_argument('-oldhashes', action='store', default=None, metavar='LMHASH:NTHASH', help='old NTLM hashes, format is LMHASH:NTHASH (the user will be asked to change their password at next logon)')
+
 	group = parser.add_argument_group('authentication')
 	group.add_argument('-hashes', action='store', default=None, metavar='LMHASH:NTHASH', help='NTLM hashes, format is LMHASH:NTHASH')
+	group.add_argument('-no-pass', action="store_true", help='don\'t ask for password (useful for -k)')
+	group.add_argument('-k', action="store_true", help='Use Kerberos authentication. Grabs credentials from ccache file (KRB5CCNAME) based on target parameters. If valid credentials cannot be found, it will use the ones specified in the command line')
+	group.add_argument('-aesKey', action="store", metavar = "hex key", help='AES key to use for Kerberos Authentication (128 or 256 bits)')
```

Comment:
- aesKey is only being used to set doKerberos flag
- The actual aesKey value is not used when setting credentials of the SMBTransport: [smbpasswd.py#L67-L70](https://github.com/fortra/impacket/blob/bbcd2d8f6bfeb7e0eb046f0fe742c689d4d03bda/examples/smbpasswd.py#L67-L70)